dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/ChatHistory.cs

```diff
@@ -124,6 +125,36 @@ public void AddRange(IEnumerable<ChatMessageContent> items)
         this._messages.AddRange(items);
     }
 
+    /// <summary>
+    /// Add a message to the chat history at the end of the streamed message
+    /// </summary>
+    /// <typeparam name="T">Specific chat message content type</typeparam>
+    /// <param name="streamingMessageContents"><see cref="IAsyncEnumerator{T}"/> list of streaming message contents</param>
+    /// <returns>Returns the original streaming results with some message processing</returns>
+    /// <exception cref="InvalidOperationException"></exception>
```

Comment:
It doesn't look like this method throws `InvalidOperationException` but if it's happening somewhere down the stack, it's probably worth to mention it in what cases this exception is thrown.

Reply:
Leftover, it has, but I changed and forgot to remove the description, thanks.